airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/write/WriteOperation.kt

```diff
@@ -23,18 +23,29 @@ import kotlinx.coroutines.runBlocking
 @Singleton
 @Requires(property = Operation.PROPERTY, value = "write")
 class WriteOperation(
-    private val inputConsumer: InputConsumer<DestinationMessage>,
+    private val inputConsumer: InputConsumer<*>,
     private val taskLauncher: TaskLauncher,
     private val taskRunner: TaskRunner
 ) : Operation {
+    val log = KotlinLogging.logger {}
+
     override fun execute() {
         runBlocking {
             launch { inputConsumer.run() }
 
             launch { taskLauncher.start() }
 
-            launch { taskRunner.run() }
+            launch {
+                try {
+                    taskRunner.run()
+                } catch (t: Throwable) {
+                    launch { taskRunner.run() }
```

Comment:
@johnny-schmidt is the intent here to continue executing other streams' tasks, even if one stream encounters an error? (I'm guessing wildly, this is basically my first time looking at this region of code)

if yes: the `.run()` inside the catch clause probably also needs to have the equivalent catch+rerun logic?

Reply:
The point is to enable the launcher to dispatch tasks that cleanup after failure, not to keep things running. So a failure at this stage would be a hard failure.